src/linalg.jl

```diff
@@ -365,6 +365,68 @@ const _length_limit = Length(200)
     end
 end
 
+@inline kron(a::StaticVector, b::StaticVector) = _kron_vec_x_vec(_length_limit, Size(a), Size(b), a, b)
+@generated function _kron_vec_x_vec(::Length{length_limit}, ::Size{SA}, ::Size{SB}, a, b) where {length_limit,SA,SB}
+    outsize = SA .* SB
+    if prod(outsize) > length_limit
+        return :( SizedVector{$(outsize[1])}( kron(drop_sdims(a), drop_sdims(b)) ) )
+    end
+    rows = [:(vcat($([:(a[$(LinearIndices(SA)[i])]*b) for i=1:SA[1]]...)))]
```

Comment:
What's the purpose of the outermost array?

I'm curious about how good (performant) `vcat` is with four or more arguments.

Reply:
Is `vcat` generally slower than `hcat`? I was trying to imitate what was already done for the `kron(StaticMatrix, StaticMatrix)`.

Reply:
Now that you pointed it out I realize that the outer array serves no purpose. Its an artifact of my attempt at molding the code that was already there for the Kronecker product between two matrices. I've removed it from other parts of the code as well.